test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommandTests/Utility/AuditUtilityTests.cs

```diff
@@ -427,6 +437,7 @@ await createGraphTasks[1]
 
         // Act
         var audit = new AuditUtility(
+            downloadDependencies: new List<DownloadDependency>(),
```

Comment:
```suggestion
            downloadDependencies: Enumerable.Empty<DownloadDependency>(),
```

I don't remember if there is an `Enumerable.Empty`, but considering `AuditUtility` takes in an `IEnumerable`, there are several zero alloc options, like `Array.Empty<T>()`.  I wouldn't be surprised if simply putting `[]` lets the compiler auto-detect its own choice for a zero alloc zero-length collection.